helpers/fetch-issues.ts

```diff
@@ -0,0 +1,72 @@
+import fs from "fs"
+import path from "path"
+
+import { constructRepoQueries } from "../graphql/queries/get-issues"
+import { getClient } from "../lib/apollo-client"
+import projects from "../public/open-source-projects/index.json"
+import { Issue, Projects } from "../types"
+
+type ProjectIssue = {
+    url: string
+    publishedAt: string
+    title: string
+    labels: string[]
+}
+
+const labels = ["good first issue", "bug", "help wanted"]
+const states = ["OPEN"]
+
+const projectRepoMetadata = Object.entries(projects as Projects).map(
+    ([_, project]) => ({
+        name: project.name,
+        owner: project.org,
+        languages: project.lang
+    })
+)
+const repoWithLabelsAndStates = projectRepoMetadata.map((repo) => ({
+    ...repo,
+    labels,
+    states
+}))
+
+const fetchAndSaveIssues = async () => {
+    try {
+        const apolloClient = getClient()
+        const query = constructRepoQueries(repoWithLabelsAndStates)
+        console.log("Fetching issues...")
+        const { data } = await apolloClient.query({ query })
+        for (const [key, issuesData] of Object.entries(data)) {
+            if (!issuesData) continue
+            const issues = (issuesData as any)?.issues?.edges as any[]
+            const projectIssues: ProjectIssue[] = []
+            issues.map((edge: { node: Issue }) => {
+                projectIssues.push({
+                    ...edge.node,
+                    labels: edge.node.labels.edges.map(
+                        (label: { node: { name: string } }) => label.node.name
+                    )
+                })
+            })
+            const repoPath = `public/open-source-projects/issues/${key}/index.json`
```

Comment:
shouldn't the format be `org:repo`.

Is there a specific reason why we are saving only repo at a directory level?

Reply:
Because it a lot simpler? I don't see any benefits of making it `org:repo`. In any case, what benefits do you envision we'd get from using `org:repo` format?